corehq/apps/locations/dbaccessors.py

```diff
@@ -91,6 +91,7 @@ def get_users_location_ids(domain, user_ids):
 
 
 def user_ids_at_locations(location_ids):
```

Comment:
what if you renamed this function `mobile_user_ids_at_locations` to be explicit

Reply:
True.